lua/plenary/scandir.lua

```diff
@@ -189,7 +189,7 @@ local conv_to_octal = function(nr)
 end
 
 local type_tbl = { [1]  = 'p', [2]  = 'c', [4]  = 'd', [6]  = 'b', [10] = '-', [12] = 'l', [14] = 's' }
-local permissions_tbl = { '--x', '-w-', '-wx', 'r--', 'r-x', 'rw-', 'rwx' }
+local permissions_tbl = { [0] = '---', '--x', '-w-', '-wx', 'r--', 'r-x', 'rw-', 'rwx' }
```

Comment:
LUL imagine using a language that has 0 indexing

Reply:
🤣